### PR TITLE
Update to be in sync with lf-5.15.32-2.0.0

### PR DESCRIPTION
**Pull request by angolini**
Please let me know if there is any missing machine.

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Curious why CONFIG_TLS is now enabled, and if we should just make that a generic option across machines.

File: bsp/imx/imx8mmevk.cfg [hunk 1]
```diff
@@ -87,7 +90,6 @@ CONFIG_GPIOLIB=y
 CONFIG_GPIO_PCA953X=y
 CONFIG_GPIO_PCA953X_IRQ=y
 CONFIG_GPIO_PL061=y
-CONFIG_GPIO_SYSFS=y
```
Comment (ricardosalveti) [inline bsp/imx/imx8mmevk.cfg:90]: Not sure why this was removed here.

File: bsp/imx/imx8mmevk.cfg [hunk 2]
```diff
@@ -223,3 +240,25 @@ CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO=y
...
+CONFIG_CRYPTO_DEV_HISI_TRNG=m
+CONFIG_CRC8=y
+CONFIG_DEBUG_INFO_REDUCED=y
+
+# enable KTLS
+CONFIG_TLS=y
+CONFIG_TLS_DEVICE=y
+
+# MGS-6358
+CONFIG_CGROUP_FREEZER=y
```
Comment (ricardosalveti) [inline bsp/imx/imx8mmevk.cfg:263]: This is already provided by cgroups.

features/cgroups/cgroups.cfg:CONFIG_CGROUP_FREEZER=y

File: bsp/imx/imx8mmevk.cfg [hunk 3]
```diff
@@ -223,3 +240,25 @@ CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO=y
...
+CONFIG_SQUASHFS_XZ=y
```
Comment (ricardosalveti) [inline bsp/imx/imx8mmevk.cfg:264]: Also not needed, this is a generic option that should be enabled outside the board config.

Comment (ricardosalveti) [reply bsp/imx/imx8mmevk.cfg:264]: When needed, which I don't think it is our case.

**Comment by ricardosalveti** [comment]
CONFIG_DEBUG_INFO_REDUCED is also a generic option that should be enabled via debug-info-reduced.cfg, when needed.

**Comment by angolini** [comment]
> Curious why CONFIG_TLS is now enabled, and if we should just make that a generic option across machines.

I could not find a reason, not on commit logs, or on Linux Reference Manual, as the commit which brings this change is the defconfig dump.

Regarding it being a common config. Would it e be a new file under `features` dir?

@ricardosalveti

**Comment by ricardosalveti** [comment]
> Regarding it being a common config. Would it e be a new file under `features` dir?

Yes, add a specific features for it and enabled as module instead of built-in, then we should be ok to have it as part of standard (since this is not specific to any hw).

File: features/tls/transport-layer-security.cfg [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# enable KTLS
```
Comment (ricardosalveti) [inline features/tls/transport-layer-security.cfg:1]: We need a scc file for it as well, and to include it as part of standard.

**Review by ricardosalveti — APPROVED** [review]
LGTM, please push the same changes to master once merged.

**Pull request merged**